vendor.conf

```diff
@@ -26,6 +26,8 @@ github.com/RackSec/srslog                           a4725f04ec91af1a91b380da679d
 github.com/imdario/mergo                            7c29201646fa3de8506f701213473dd407f19646 # v0.3.7
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
+github.com/tklauser/go-sysconf                      d90dfe8b2f4be4b0a4df5db89401b0ba74fe24d9 # sysconf for Go, without using cgo
+
```

Comment:
please put in a same commit with associated modification inside dockerd so that we can see why this additional dependency is required.

Reply:
just the vendor.conf changes or along with the vendor folder changes too? That is going to be a bit huge then.

Reply:
Vendor folder too. 
It's already a huge diff.

Reply:
okay, that isn't a conventional way I have seen in the docker commit history. But going for it anyway :)